terraform/files/bin/create_cluster.sh

```diff
@@ -215,7 +216,7 @@ if test "$DEPLOY_NGINX_INGRESS" = "true" -o "${DEPLOY_NGINX_INGRESS:0:1}" = "v";
 fi
 
 echo "# Wait for control plane of ${CLUSTER_NAME}"
-kubectl config use-context kind-kind
+~/bin/mng_cluster_ns.sh
```

Comment:
`/home/ubuntu/bin/create_cluster.sh: line 219: /home/ubuntu/bin/mng_cluster_ns.sh: No such file or directory`

Reply:
What is convention, should we call it `.inc` or `.sh`?

Reply:
Also, shouldn't be `CREATE_NEW_NAMESPACE=false` included here to avoid the error of created namespace in the logs?

Reply:
It seems to me that `.sh` are "public" facing scripts and `.inc` are private scripts.

Reply:
As cluster should already exist at this point it should not create a new namespace.

Reply:
Ok sorry I understand now